stdlib/ntpath.pyi

```diff
@@ -1,97 +1,45 @@
-import os
 import sys
-from _typeshed import AnyPath, BytesPath, StrPath
-from genericpath import exists as exists
+from _typeshed import BytesPath, StrPath
+from genericpath import *
 from os import PathLike
-from os.path import commonpath as commonpath, commonprefix as commonprefix, lexists as lexists
-from typing import AnyStr, Optional, Tuple, TypeVar, overload
+from os.path import (
+    abspath as abspath,
+    altsep as altsep,
+    basename as basename,
+    commonpath as commonpath,
+    curdir as curdir,
+    defpath as defpath,
+    devnull as devnull,
+    dirname as dirname,
+    expanduser as expanduser,
+    expandvars as expandvars,
+    extsep as extsep,
+    isabs as isabs,
+    islink as islink,
+    ismount as ismount,
+    lexists as lexists,
+    normcase as normcase,
+    normpath as normpath,
+    pardir as pardir,
+    pathsep as pathsep,
+    relpath as relpath,
+    sep as sep,
+    split as split,
+    splitdrive as splitdrive,
+    splitext as splitext,
+    supports_unicode_filenames as supports_unicode_filenames,
+)
+from typing import AnyStr, overload
 
-_T = TypeVar("_T")
-
-# ----- os.path variables -----
-supports_unicode_filenames: bool
-# aliases (also in os)
-curdir: str
-pardir: str
-sep: str
-if sys.platform == "win32":
-    altsep: str
-else:
-    altsep: Optional[str]
```

Comment:
Now `altsep` is `Optional[str]` for all platforms. For Windows, it should be `str`, so that you don't have to `assert os.altsep is not None` every time you use it in windows-only code.

Reply:
Actually, `ntpath.altsep` is never `None`, because `os.path` is just an alias for `ntpath` on windows and `posixpath` on posix.

Reply:
Sorry, is there a change to be made? The definition of altsep in path is:
```
if sys.platform == "win32":
    altsep: str
else:
    altsep: Optional[str]
```
so it should be the same, right?

Reply:
Good catch. Maybe doing this re-exporting backwards isn't wise..